.drone.yml

```diff
@@ -17069,6 +17069,6 @@ image_pull_secrets:
 - DOCKERHUB_CREDENTIALS
 ---
 kind: signature
-hmac: a0cd6e281f864553c31bf642861283cdfa54275907721fcbd1949b3ca239a3e1
+hmac: 54c72179e16cb5f8c2c7bfecdb28ce2bfcbe73c671b555b610a9e353f656abd3
```

Comment:
Why did this change?

Reply:
This is a signature of the file. It changes any time there are changes to .drone.yml.